nomad-core/Cargo.toml

```diff
@@ -25,6 +25,12 @@ rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb" }
 prometheus = "0.12.0"
 bytes = { version = "1", features = ["serde"]}
 num = {version="0", features=["serde"]}
+rusoto_core = "0.47.0"
```

Comment:
I don't like moving the aws deps into core 🤔  where's a better place to put these in the future?

Reply:
Yeah only reason this went here is so we could build a `Signers` from `SignerConf`. Best solution is probably to keep `SignerExt` in `nomad-core` and move `Signers` (enum over Aws and Local types) into `nomad-base`

Reply:
Well turns out `ethers` technically imports `rusoto_core` anyways because it has `AwsSigner` types. So regardless, aws has to be a dependency of `nomad-core` if we import ethers

Reply:
ack oh well